pkg/console/service/service.go

```diff
@@ -289,9 +290,9 @@ func (s *service) requestToken(ctx context.Context, serviceAccountNamespace stri
 		metav1.CreateOptions{},
 	)
 	if err != nil {
-		return "", err
+		return authnv1.TokenRequestStatus{}, err
```

Comment:
Make the first return value a ptr and return `nil` on error?

Reply:
done